src/main/java/chess/dao/PieceDao.java

```diff
@@ -19,9 +19,13 @@
 @Repository
 public class PieceDao {
 
-    @Autowired
     private JdbcTemplate jdbcTemplate;
```

Comment:
```suggestion
    private final JdbcTemplate jdbcTemplate;
```

`GameDao`에서는 이미 `final` 키워드를 사용하고 있네요 😊

Reply:
수정했습니다..!